From randy.dunlap@oracle.com  Wed Mar 18 21:50:02 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Wed, 18 Mar 2009 10:00:16 -0700
Subject: Staging: line6: fix build error, select SND_RAWMIDI
To: Stephen Rothwell <sfr@canb.auug.org.au>
Cc: Greg KH <gregkh@suse.de>
Message-ID: <49C128A0.7030805@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

line6 needs to select SND_RAWMIDI, like many other drivers do.

ERROR: "snd_rawmidi_set_ops" [drivers/staging/line6/line6usb.ko] undefined!
ERROR: "snd_rawmidi_new" [drivers/staging/line6/line6usb.ko] undefined!
ERROR: "snd_rawmidi_transmit_peek" [drivers/staging/line6/line6usb.ko] undefined!
ERROR: "snd_rawmidi_transmit_ack" [drivers/staging/line6/line6usb.ko] undefined!
ERROR: "snd_rawmidi_receive" [drivers/staging/line6/line6usb.ko] undefined!

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/line6/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/line6/Kconfig
+++ b/drivers/staging/line6/Kconfig
@@ -1,6 +1,7 @@
 config LINE6_USB
 	tristate "Line6 USB support"
 	depends on USB && SND
+	select SND_RAWMIDI
 	help
 	  This is a driver for the guitar amp, cab, and effects modeller
 	  PODxt Pro by Line6 (and similar devices), supporting the
